src/table/styled-components.js

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index.js';
+import type {SharedStylePropsT} from './types.js';
+import {Grid} from 'react-virtualized';
+
+/**
+ * Main component container element
+ */
+export const Root = styled('div', ({$theme}: SharedStylePropsT) => {
+  const {borders} = $theme;
+  return {
+    ...borders.border300,
+    borderRadius: borders.radius200,
+  };
+});
+
+export const Head = styled(Grid, ({$theme}: SharedStylePropsT) => {
+  const {lighting} = $theme;
+  return {
+    textAlign: 'left',
+    boxShadow: lighting.shadow400,
+  };
+});
+
+export const HeadCell = styled('div', ({$theme}: SharedStylePropsT) => {
+  const {borders, sizing, typography} = $theme;
+  return {
+    padding: `${sizing.scale500} ${sizing.scale0} ${sizing.scale500} ${
+      sizing.scale600
+    }`,
+    ...typography.font350,
+    boxSizing: 'border-box',
+    height: '100%',
+    width: '100%',
+    ...borders.border400,
+    borderLeft: 'none',
+    borderTop: 'none',
+    borderBottom: 'none',
+  };
+});
+export const Body = styled(Grid, (props: SharedStylePropsT) => {
+  return {};
+});
+export const Cell = styled('div', ({$theme}: SharedStylePropsT) => {
```

Comment:
should we use table semantics here? so in the example of the cell, it should be `td`

Reply:
oh, I've missed that

Reply:
in that case, should we add ARIA roles? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role

Reply:
added aria roles in 7cb20ab42e612a3b2e853b8e4d7d639e0c15b03f

Reply:
Using `grid` role instead of table becuase

> If a table maintains a selection state, has two-dimensional navigation, or allows the user to rearrange cell order use grid or treegrid instead.